src/java.xml/share/classes/jdk/xml/internal/jdkcatalog/w3c/dtd/schema10/XMLSchema.dtd

```diff
@@ -61,14 +57,11 @@
 
 <!-- Customisation entities for the ATTLIST of each element type.
      Define one of these if your schema takes advantage of the
-     anyAttribute='##other' in the
-     schema for schema documents -->
+     anyAttribute='##other' in the schema for schemas -->
```

Comment:
Some changes seem to be present to minimize differences from the official versions.
Keeping the split line from the original would show fewer differences.